client/src/lib/const.ts

```diff
@@ -0,0 +1,3 @@
+// Between 8-30, at least one uppercase letter, one lowercase letter, one number and one special character
+export const PASSWORD_REGEXP =
+	/^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,30}$/;
```

Comment:
Uh, this looks ugly AF.
I need to check, how libraries do the validation.

Reply:
I did not see any validator for passwords, I would be possible to write some validators for _svelte-forms_, something more granular, like `oneLowerCase()`, `oneUpperCase()`, `oneNumber()` and so on with smaller regexes tucked inside them, but it would be nice if I did not have to and just import it from somewhere 😄 . But your choice boss. Please do tell if you find something.

Reply:
I don't want to be a boss of any kind. Just afraid of long regexes, that I don't understand.
Go with this for now, will adjust later if needed.